src/core/tsi/ssl_transport_security.h

```diff
@@ -209,19 +209,13 @@ tsi_result tsi_create_ssl_client_handshaker_factory_with_options(
   - server_name_indication indicates the name of the server the client is
     trying to connect to which will be relayed to the server using the SNI
     extension.
+  - network_bio_buf_size and ssl_bio_buf_size represent BIO pair buffers used in
```

Comment:
Wait... are we not actually passing in a non-zero value here from any of the callers?  If not, why do we need to add these parameters in the first place?

Reply:
We passed non-zero values to the callers in `ssl_transport_security_test.cc`. For non-test callers, the passed in values are always zero.

Reply:
Ah, okay.

In the future, when we convert this code to a C++ API, we should consider making an args struct, so that callers need to specify only the parameters that are getting non-default values.  But this is fine for now.